numba/unicode.py

```diff
@@ -442,8 +454,18 @@ def _codepoint_is_ascii(ch):
     return ch < 128
 
 
+@register_jitable
+def normalize_slice_arg(arg, slice_len, default):
+    if arg is None:
+        return default
+    elif arg < 0:
+        return max( arg + slice_len, default)
```

Comment:
```suggestion
        return max(arg + slice_len, default)
```

Reply:
oh thanks for bringing that to my notice! Will fix it